src/main/java/frc/robot/constants/DriveConstants.java

```diff
@@ -1,9 +1,175 @@
+    public static final double kDriveKSFrontLeft = 0.61534;
+    public static final double kDriveKVFrontLeft = 4.45071;
+    public static final double kDrivePFrontLeft = 2;
+    public static final double kDriveIFrontLeft = 0;
+    public static final double kDriveDFrontLeft = 0;
+    public static final double kSteerKSFrontLeft = 1;
+    public static final double kSteerKVFrontLeft = 0.5;
+    public static final double kSteerPFrontLeft = 12;
+    public static final double kSteerIFrontLeft = 0;
+    public static final double kSteerDFrontLeft = 0;
+    
+    public static final int kDriveFrontRight = 4; 
+    public static final int kSteerFrontRight = 5;
+    public static final int kEncoderFrontRight = 6; 
+    public static final double kSteerOffsetFrontRight = -2.764+Math.PI;
+    public static final double kDriveKSFrontRight = 0.61283;
+    public static final double kDriveKVFrontRight = 4.45431;
+    public static final double kDrivePFrontRight = 2;
+    public static final double kDriveIFrontRight = 0;
+    public static final double kDriveDFrontRight = 0;
+    public static final double kSteerKSFrontRight = 1;
+    public static final double kSteerKVFrontRight = 0.5;
+    public static final double kSteerPFrontRight = 12;
+    public static final double kSteerIFrontRight = 0;
+    public static final double kSteerDFrontRight = 0;
+    
+    public static final int kDriveBackLeft = 7;
+    public static final int kSteerBackLeft = 8;
+    public static final int kEncoderBackLeft = 9; 
+    public static final double kSteerOffsetBackLeft = 0;
+    public static final double kDriveKSBackLeft = 0.54150;
+    public static final double kDriveKVBackLeft = 4.53909;
+    public static final double kDrivePBackLeft = 2;
+    public static final double kDriveIBackLeft = 0;
+    public static final double kDriveDBackLeft = 0;
+    public static final double kSteerKSBackLeft = 1;
+    public static final double kSteerKVBackLeft = 0.5;
+    public static final double kSteerPBackLeft = 12;
+    public static final double kSteerIBackLeft = 0;
+    public static final double kSteerDBackLeft = 0;
+    
+    public static final int kDriveBackRight = 10;
+    public static final int kSteerBackRight = 11;
+    public static final int kEncoderBackRight = 12; 
+    public static final double kSteerOffsetBackRight = 2.73;
+    public static final double kDriveKSBackRight = 0.49599;
+    public static final double kDriveKVBackRight = 4.75897;
+    public static final double kDrivePBackRight = 2.2;
+    public static final double kDriveIBackRight = 0;
+    public static final double kDriveDBackRight = 0;
+    public static final double kSteerKSBackRight = 1;
+    public static final double kSteerKVBackRight = 0.5;
+    public static final double kSteerPBackRight = 12;
+    public static final double kSteerIBackRight = 0;
+    public static final double kSteerDBackRight = 0;
+    
+  }
+  
+  // PID
+  // Drive
+  public static final double kDriveP = 0;
```

Comment:
Seems like these variables and below are already defined above in the CompBotConstants or TestDriveConstants?